Makefile

```diff
@@ -0,0 +1,15 @@
+init:
+	pip install --upgrade pipenv
+	pipenv install --dev
```

Comment:
Do you mind if we remove the `init` target in favor of just having this duplicated in `.travis.yml` and the `README.md`? I feel like many Python people will already have have a `pipenv` available, and there are other recommended ways of installing it (e.g., I used Homebrew as recommended on their page).

Reply:
👍